src/components/offers/list/HrOffersView.js

```diff
@@ -0,0 +1,19 @@
+import {offersAPI} from "../../../utils/apis/OfferApi";
+import {hrOfferButtons} from "../HrOfferButtons";
+import {theme} from "../../../test/utils/theme";
+import {OffersView} from "./OffersView";
+import React from "react";
+import {Link} from "react-router-dom";
+import {Button} from "@material-ui/core";
+
+const HrOffersView = () => <div style={{width: "90%", marginRight: "auto", marginLeft: "auto"}}>
+    <Link to="/hr/offers/add">
```

Comment:
To miało być zamiast przycisku 'offers' w nawigacji - dla robaka przekierowuje na wszystkie, dla HRa na jego

Reply:
Imo zmienianie zachowanie guzika na navbarze w zależności od typu użytkownika jest confusing. Dowolny, nawet niezalogowany gość dostaje tam wszystkie oferty jakie są dostępne, a nagle ktoś loguje się na konto hra i dostaje widok ograniczony do swoich ofert (już pomijając fakt, że nie jest w stanie w tym momecie dostać się do tego widoku wszystkich ofert). Wrzucenie przycisku prowadzącego do tego widoku wydaje mi się bardziej naturalne w rozwijanym menu, które służy stricte do rzeczy typowych dla danego użytkownika.